samples/src/test/java/com/example/bigquery/ExtractTableToJSONIT.java

```diff
@@ -38,39 +26,9 @@ public void tearDown() {
 
   @Test
   public void testExtractTableToJSON() {
-    String generatedDatasetName = RemoteBigQueryHelper.generateDatasetName();
-
-    // Create a new dataset to create a new table in
-    CreateDataset.createDataset(generatedDatasetName);
-
-    // Create a new table to extract to GCS for
-    String tableName = "my_table_name";
-    Schema schema =
-        Schema.of(
-            Field.of("stringField", LegacySQLTypeName.STRING),
-            Field.of("booleanField", LegacySQLTypeName.BOOLEAN));
-    Table table = createTableHelper(generatedDatasetName, tableName, schema);
-
     // Extract table content to GCS in CSV format
-    ExtractTableToJSON.extractTableToJSON(table, "CSV", "gs://my-bucket/extractTest.csv");
+    ExtractTableToJSON.extractTableToJSON(
+        "bigquery-public-data", "samples", "shakespeare", "gs://my-bucket/extractTest.csv");
```

Comment:
Is `my-bucket` a valid bucket name for these tests? Should these values be passed in via environment variables instead? (if yes, please add a check that the environment variables are set or throw an error if not)